userspace/libsinsp/version.h

```diff
@@ -46,8 +46,65 @@ class sinsp_version
 			+ "." + std::to_string(m_version_patch);
 	}
 
-	inline bool check(const sinsp_version &requested) const
+	bool operator<(sinsp_version const& right) const
```

Comment:
do these API need to follow the Semver convention? If yes there is something strange...according to this logic `6.1.3` è minore di `3.2.0` fo example

Reply:
yes you are right, my bad!